crates/bevy_ui/src/render/mod.rs

```diff
@@ -938,7 +984,11 @@ pub mod shader_flags {
     pub const TEXTURED: u32 = 1;
     /// Ordering: top left, top right, bottom right, bottom left.
     pub const CORNERS: [u32; 4] = [0, 2, 2 | 4, 4];
-    pub const BORDER: u32 = 8;
+    pub const BORDER_LEFT: u32 = 8;
+    pub const BORDER_TOP: u32 = 16;
+    pub const BORDER_RIGHT: u32 = 32;
+    pub const BORDER_BOTTOM: u32 = 64;
+    pub const BORDER_ALL: u32 = BORDER_LEFT + BORDER_TOP + BORDER_RIGHT + BORDER_BOTTOM;
```

Comment:
There will probably be other uses for these flags in the future, so it might be good to give them more general names like `LEFT_EDGE` or just `LEFT` etc. The order of CSS edge values always goes "Top → Right → Bottom → Left" so I'd like to swap around the constants to that order as well.

This a bit fussy though, and it's not a problem at the moment with this PR. If it doesn't bother you, I can clean it up later sometime in a followup.